threescale/api.go

```diff
@@ -94,6 +113,34 @@ func (c *Client) doAuthorizeReq(req *http.Request, extensions Extensions) (*Auth
 	return c.handleAuthorizeExtensions(resp, response, extensions), nil
 }
 
+func (c *Client) doReportReq(req *http.Request, values url.Values) (*ReportResponse, error) {
+	req.URL.RawQuery = values.Encode()
+	req.Header.Set("Accept", "application/xml")
+
+	resp, err := c.httpClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != http.StatusAccepted {
```

Comment:
Best to take any 2XX value in these cases, not just 202.

Reply:
yep had thought about that. Will fix